src/Autosuggest.js

```diff
@@ -208,13 +237,19 @@ export default class Autosuggest extends Component {
 
   onInputBlur() {
     this.setSuggestionsState(null);
+    this.onSuggestionUnfocused();
+    lastFocusedSuggestion = null;
   }
 
   onSuggestionMouseEnter(sectionIndex, suggestionIndex) {
+    let suggestion = this.getSuggestion(sectionIndex, suggestionIndex);
     this.setState({
       focusedSectionIndex: sectionIndex,
       focusedSuggestionIndex: suggestionIndex
     });
+
+    this.props.onSuggestionFocused(suggestion);
+    lastFocusedSuggestion = suggestion;
```

Comment:
Yeah, I feel like we better have `this.onSuggestionFocused()` (similarly to `this.onSuggestionUnfocused()`) and set `lastFocusedSuggestion` there.